tests/network/vmi_networking.go

```diff
@@ -653,10 +653,14 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		}
 
 		fedoraMasqueradeVMI := func(ports []v1.Port, ipv6NetworkCIDR string) (*v1.VirtualMachineInstance, error) {
+			if ipv6NetworkCIDR == "" {
+				ipv6NetworkCIDR = libnet.DefaultIPv6CIDR
+			}
 			networkData, err := libnet.NewNetworkData(
 				libnet.WithEthernet("eth0",
 					libnet.WithDHCP4Enabled(),
-					libnet.WithDHCP6Enabled(),
+					libnet.WithAddresses(ipv6NetworkCIDR),
```

Comment:
The test fails since the CIDR is used here instead of the address.

Reply:
I pushed a [fix](https://github.com/kubevirt/kubevirt/compare/31fe1457f726ed382cb76043022571d1f9d7dbb8..b479dd634471661a111f88dcb7a7395c16848d7b), hopefully.